File: MySolutions/Python/q1_TwoSum.py
#TwoSum - Naive Approach

class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        indices=[]
        for i in range(len(nums)):
            for j in range(i+1,len(nums)):
                if target-nums[i]==nums[j]:
                        indices.extend([i,j])
        
        return indices

File: MySolutions/Cpp/q1_TwoSum.cpp
#1 Two Sum

class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> res={};
        for(int i=0;i<nums.size();i++){
            int num1=nums[i];
            for(int j=i+1;j<nums.size();j++){
                if(nums[j]==target-nums[i]){
                    res.push_back(i);
                    res.push_back(j);
                    break;
                }
            }
        }
        return res;
    }
};
